cmd/user_service/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+	retty3ServiceUser "github.com/RettyInc/retty3-service-user"
```

Comment:
package名はcamelCaseにしない習慣ですね
```suggestion
	retty3serviceuser "github.com/RettyInc/retty3-service-user"
```
それとpackage名をわざわざ変えなくても良いのではないでしょうか

Reply:
確かに, `userservice`  にわざわざする理由がないのと, ディレクトリ名と一致したほうが分かりやすそうなのでそうします 🙇